src/molecules/TrustedClients/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Box,
+  Image,
+  ListItem,
+  List,
+  Text,
+} from '../../atoms';
+
+// TODO: remove this once kamesh makes header component
+const H4 = styled.h4`
```

Comment:
component is already there
https://github.com/Codebrahma/serverless-design-system/blob/master/docs/src/heading.js#L9